jobs/models.py

```diff
@@ -1,3 +1,62 @@
+from django.utils import timezone
+
 from django.db import models
+from django_countries.fields import CountryField
+from djangogirls import settings
```

Comment:
from core.models import User

(normally, if we were using the standard django auth, we'd do 
from django.contrib.auth.models import User
however, since the User model has been overwritten in core.models, we need to get the User from there instead)